components/ScheduleHome.vue

```diff
@@ -64,6 +98,9 @@ export default {
       } else {
         return this.nowPlaying?.now_playing?.song.artist
```

Comment:
This conditional operator is actually pretty cool!  

But if it returns none then how do we handle it? Since it is already in the `else` statement part of execution

Reply:
it is the new Nullish coalescing operator, really makes things easier for handling unavailable data.
if none is returning it just gives empty string I think, but if you can think some edge cases feel free to tell

Reply:
You use it in `showAirCheck` for logic to verify there is some show on air. That is why I asked how we handle it. Is an empty string ~trueish?